src/main/java/dev/codesoapbox/dummy4j/AbstractDummy4jBuilder.java

```diff
@@ -34,5 +36,33 @@ public T paths(List<String> paths) {
         return self();
     }
 
+    /**
+     * @since SNAPSHOT
+     */
+    public T locale(String... locale) {
+        if (theOnlyElementIsNull(locale)) {
+            self().locale = new ArrayList<>();
+        } else {
+            self().locale = Arrays.asList(locale);
+        }
+        return self();
+    }
+
+    private boolean theOnlyElementIsNull(String[] elements) {
+        return elements.length == 1 && elements[0] == null;
```

Comment:
Not sure what to rename it but you could encapsulate even more logic in the method:
```java
private List<String> varArgsToList() {
  if(elements.length == 1 && elements[0] == null) {
    return emptyList();
  }
  return Arrays.asList(elements);
}
```
Then:
```java
public T locale(String... locale) {
  self().locale = varArgsToList(locale);
  return self();
}
```